doc/source/reference/random/multithreading.rst

```diff
@@ -107,3 +103,11 @@ that does not use an existing array due to array creation overhead.
        ...: %timeit rg.standard_normal(10000000)
 
     Out[6]: 125 ms ± 309 µs per loop (mean ± std. dev. of 7 runs, 10 loops each)
+
+Note that if `threads` is not set by the user, it will be determined by the architecture
```

Comment:
I recommend being a bit more explicit here by replacing "the architecture" with `multiprocessing.cpu_count`.

Reply:
Sounds good! I wasn't sure on how explicit I should be.